gui/public/sp_load_draws.py

```diff
@@ -0,0 +1,67 @@
+# Used in pyodideWorker for running analysis.py
+
+from typing import List
+import numpy as np
+
+
+class DrawsObject:
+    def __init__(self, sp_data):
+        if "sampling" not in sp_data:
+            raise ValueError("sampling key not found in sp_data")
+        sampling: dict = sp_data["sampling"]
+        self._draws: List[List[float]] = sampling["draws"]
```

Comment:
I think it would make sense to have the internal representation use numpy starting in the constructor, rather than doing it on demand later. I think all the ways we want to expose it will prefer the data already be an array

Reply:
Right now the other parts of this class assume it's a Python list. Should we rewrite it to use numpy arrays? I'm inclined to leave it as is, but if you think there's an important reason.

Reply:
I can't speak for Brian here, but one reason that occurs to me would be redundant memory use: if we store as both a list and an NDArray, we're storing the data twice, which could get burdensome for large draw counts.

Within the class, it looks like most of what we would be doing with the list would work for an `ndarray` with some fairly small modifications. Are there other places it's used that would be of concern?

Reply:
Fair point about the memory. If one of you wants to rewrite it using numpy internally it's fine with me, but I prefer it as is so I wouldn't want to go to the effort myself.